src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -244,6 +251,39 @@ function MultiSelect<T>({
         setSelectedItems(externalSelectedItems);
     }, [externalSelectedItems?.map((item) => itemId(item)).join("|")]);
 
+    React.useEffect(() => {
+        if (!maxHeight) return;
```

Comment:
A test for allowed range is missing, values over 100 would also not be make sense for the calculation.